To explain the new Download button (#58)
akeneo/sfcc-helpcenter: content/md/cartridge-installation/download-cartridge.md

**Pre-requisites:**

> First of all, you need to access the [**Akeneo partner Portal**](https://partners.akeneo.com/) and select the project on which the cartridge has to be installed for.

Reviewer: ```suggestion
First of all, you need to access the [**Akeneo partner Portal**](https://partners.akeneo.com/) and select the project on which the cartridge has to be installed.
```

:::info

> Your CSM has to activate the connector access for this project to see the Download link.

Reviewer: ```suggestion
Your CSM has to activate the connector access for this project for you to able able to see the download link.
```

:::

> Once you have clicked, you will have to select the archive version that you want to install.

Reviewer: ```suggestion
Once you clicked, you will have to select the archive version that you want to install.
```

![SFCC download button](./img/Download-cartridge-button.gif)

:::info

> We strongly recommended to get the last version

Reviewer: ```suggestion
We strongly recommend you to get the latest version.
```
